Youtube-project.html

```diff
@@ -0,0 +1,578 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <!--Calling style,icons and fonts libraries-->
+  <script src="https://kit.fontawesome.com/0007bfca3c.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" type="text/css" href="style.css">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <!-- Adding tab title-->
+  <title>Youtube</title>
+  <meta charset="UTF-8"> 
+</head>
+
+<body>
+  <nav id="navbar" class="navstyle">
+    <!--<div id="container" class="navstyle">-->
+      <!--Left container-->
+      <div class="navstyle">
+        <div class="navstyle">
+          <div id="menu">
+            <div class="menu-bars"></div>
+            <div class="menu-bars"></div>
+            <div class="menu-bars"></div>
+          </div>
+        </div>
+        <div class="navstyle">
+          <i class="fab fa-youtube fa-2x"></i>
+          <span id="youtube" class="navstyle">YouTube</span>
+        </div>    
+      </div>  
+      <!--Center container-->
+      <div id="center" class="navstyle">
+        <div id="searchcont" class="navstyle">
+          <input id="searchbox" type="search" placeholder="  Search">
+          <button id="searchbutton">
+            <i class="fas fa-search"></i>
+          </button>
+        </div> 
+      </div>
+      <!--Right container-->
+      <div id="right" class="navstyle">
+        <div id="icons" class="navstyle">
+          <i class="large material-icons navstyle">video_call</i>
+          <i class="fas fa-th navstyle"></i>
+          <i class="fas fa-bell navstyle"></i>
+          <img id="userimg" src="./userimg.jpeg" alt="user image" class="navstyle">
+        </div>               
+      </div>
+    <!--</div>-->
+  </nav>
+  <main id="main">
+    <div id="columns">
+      <div id="user-column">
+        <div>
+          <!--///////////////////// PLAYER DIV.\\\\\\\\\\\\\\\\\\\\\-->
+          <div id="player-cont">
+            <div id="player">
+              <iframe id="video-player" src="https://www.youtube.com/embed/V74l_zS1x8E" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+            </div>
+          </div>
+          <!--///////////////////// INFO DIV.\\\\\\\\\\\\\\\\\\\\\-->
+          <div id="info">
+            <h1 id="video-title" class="text-style">Contributing to Open Source Part I: The Easy Way</h1> <!--youtube video title-->
+            <div id="info-icons">
+              <div id="icons-left">
+                <div class="text-style">57,638 views</div> <!--Number of views-->                
+              </div>
+              <div id="icons-right">
+                <div  class="text-style button-container">
+                  <a id="like-button" class="text-style">
+                  <div id="iconlike-cont" class="text-style user-style">
+                    <i class="fas fa-thumbs-up"></i> <!--Like button-->
```

Comment:
too many id's, actually ids are used only if you want to select elements easily with JavaScript, so that's there basic use, and not to style cause you can select whatever item in the DOM with just class selectors.